egs/ami/s5/local/tfrnnlm/run_lstm.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+mic=ihm
+ngram_order=4
```

Comment:
explain that this affects state-merging when rescoring (an approximation for speed).